apps/web/src/api/notification-templates.ts

```diff
@@ -32,7 +32,7 @@ export async function getBlueprintsGroupedByCategory(): Promise<{
   return api.get(`${BLUEPRINTS_API_URL}/v1/blueprints/group-by-category`, { absoluteUrl: true });
 }
 
-export async function getBlueprintTemplateById(id: string) {
+export async function getBlueprintTemplateById(id: string): Promise<INotificationTemplate> {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** Yay explicit types!‏